### DISCUSSION TITLE
Music mod doesn't work

### DISCUSSION THREAD

**Discussion by Midos-cmd** [thatskymod/Sky-CotL-Scripts#103] [Q&A] [answered]
The mod for playing music has probably not been working for a couple of days now. I tried updating, reinstalling everything, rebooting the phone, but it still doesn’t work. Canvas starts, all other mods work. The music mod also appears as a working mod, but when you click play music, it’s just a timer, without a melody. Before this everything was fine. I tried different tools, and all to no avail. 

It would also be cool to add a cello already, because there is already a grand piano.

**Comment by thatskymod** [comment, marked as answer]
# sMusic v1.1.4
- Updated for Sky Live v0.26.2
- Added 2 new instruments. Cello & Season of Duets grand piano
- Improved compatibility with some devices

Download:
https://github.com/thatskymod/Sky-CotL-Scripts/blob/fc40f610ccac717e972f7dc9a9ce1d4d9df3af2c/Canvas/Libs/libsmusic-1.1.4.so

**Reply by bdalrhmnghwyly79** [reply]
> # sMusic v1.1.4
> 
> - Updated for Sky Live v0.26.2
> 
> - Added 2 new instruments. Cello & Season of Duets grand piano
> 
> - Improved compatibility with some devices
> 
> 
> 
> Download:
> 
> https://github.com/thatskymod/Sky-CotL-Scripts/blob/fc40f610ccac717e972f7dc9a9ce1d4d9df3af2c/Canvas/Libs/libsmusic-1.1.4.so
